tests/tools/test_output_validation.py

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
@serra I have a PR lined up for a larger change to the unit tests to validate the output types. Perhaps we can skip this test and I will follow up with that PR.